package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "worldview",
```

Comment:
We're not installing this as a dependency, so it might not matter, but the package name [`worldview` is already taken on npm](https://www.npmjs.com/package/worldview). Could we use `nasa-worldview` instead?

Reply:
Gah, yea we better make it unique.